packages/providers/upload-local/lib/index.js

```diff
@@ -9,18 +9,11 @@ const { pipeline } = require('stream');
 const fs = require('fs');
 const path = require('path');
 const fse = require('fs-extra');
-const { PayloadTooLargeError } = require('@strapi/utils').errors;
 
 const UPLOADS_FOLDER_NAME = 'uploads';
 
 module.exports = {
-  init({ sizeLimit = 1000000 } = {}) {
-    const verifySize = (file) => {
```

Comment:
I think it's here that we should keep this code if the option exists an deprecate it actually

Reply:
About this I have 1 question.

In the docs, we mentioned the sizeLimit should be in bytes.
But the `file.size` the provider receives here is in kilobytes, so the verifySize function is not working as it should (instead of limiting to 1MB files it is limiting to 1000 MB). 

I understand changing this could be an issue for users, but should we change the docs at least?

Reply:
Yes you should change the defaults to use the new sizeCheck :) so it only shows the warning for older projects

Reply:
For the conversion pbl if we just change the doc we will still have a pbl with the default sizeLimit. I would fix the comparison and add a migration documentation for the upgrade

Reply:
👍  Makes sense. I will communicate this with the docs team and we will prepare documentation for that.

Reply:
@alexandrebodin 
We were debating with @pwizla  the need of making a migration docs for this change, and wanted to ask you Alex what do you think about it.

In the docs ,we were mentioning people to **limit the size in bytes, and this is how it will be now**. 
So do we consider this i**mportant enough to make a migration guide**?

We commented the possibility of adding a caution block in the [max file size section](https://docs.strapi.io/developer-docs/latest/plugins/upload.html#configuration). Saying in versions prior to X size limit was sin kB and now in bytes.

What do you think?

Reply:
If we consider this as a fix we can just indicate that change and how to update the config